<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/icon.png</normaloff>img/icon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.980000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(48, 38, 38);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="screen_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton0">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>520</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(9, 8, 8);
color: rgb(255, 255, 255);}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="shortcut">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equals_pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>520</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(29, 25, 26);

color: rgb(255, 255, 255);
}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="shortcut">
     <string>=, Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deci_pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>520</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(29, 25, 26);

color: rgb(255, 255, 255);
}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="shortcut">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dl_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(29, 25, 26);

color: rgb(255, 255, 255);
}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>Dark/Light</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(9, 8, 8);
color: rgb(255, 255, 255);}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="shortcut">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(9, 8, 8);
color: rgb(255, 255, 255);}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="shortcut">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>440</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(9, 8, 8);
color: rgb(255, 255, 255);}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="shortcut">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(9, 8, 8);
color: rgb(255, 255, 255);}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="shortcut">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(9, 8, 8);
color: rgb(255, 255, 255);}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="shortcut">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(9, 8, 8);
color: rgb(255, 255, 255);}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="shortcut">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(9, 8, 8);
color: rgb(255, 255, 255);}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="shortcut">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton8">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(9, 8, 8);
color: rgb(255, 255, 255);}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="shortcut">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(9, 8, 8);
color: rgb(255, 255, 255);}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="shortcut">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>440</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(29, 25, 26);

color: rgb(255, 255, 255);
}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="shortcut">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus_pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(29, 25, 26);

color: rgb(255, 255, 255);
}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="shortcut">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mul_pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(29, 25, 26);

color: rgb(255, 255, 255);
}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="shortcut">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="div_pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(29, 25, 26);

color: rgb(255, 255, 255);
}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <property name="shortcut">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="c_pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(29, 25, 26);

color: rgb(255, 255, 255);
}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
    <property name="shortcut">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ce_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(29, 25, 26);

color: rgb(255, 255, 255);
}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
    <property name="shortcut">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upon_pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>139</y>
      <width>115</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(29, 25, 26);

color: rgb(255, 255, 255);
}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>⅟x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pow_pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>139</y>
      <width>115</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(29, 25, 26);

color: rgb(255, 255, 255);
}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>xⁿ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="root_pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>139</y>
      <width>115</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(29, 25, 26);

color: rgb(255, 255, 255);
}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="percent_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>139</y>
      <width>115</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(29, 25, 26);

color: rgb(255, 255, 255);
}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
    <property name="shortcut">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(29, 25, 26);

color: rgb(255, 255, 255);
}

QPushButton::hover{

	background-color: rgb(166, 151, 155);
}

QPushButton::pressed{
background-color: rgb(166, 151, 155);
}</string>
    </property>
    <property name="text">
     <string>«</string>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
   </widget>
   <widget class="QLabel" name="screen2_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(198, 198, 198);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
